### PR TITLE
A syrupy replacement

### PR DESCRIPTION
**Pull request by wkerzendorf**
Syrupy is terribly complex for what we need. I think this would do what we want with minimal code (less than even our customization of syrupy)

### CODE REVIEW

File: tardis/tests/fixtures/regression_data.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+class RegressionData:
+    def __init__(self, request) -> None:
+        self.request = request
+        regression_data_path = request.config.getoption(
+            "--tardis-regression-data"
+        )
+        if regression_data_path is None:
+            pytest.skip("--tardis-regression-data was not specified")
+        self.regression_data_path = Path(
+            os.path.expandvars(os.path.expanduser(regression_data_path))
```
Comment (Knights-Templars) [inline tardis/tests/fixtures/regression_data.py:21]: @jvshields commented there could be a better way.

File: tardis/plasma/tests/test_hdf_plasma.py [hunk 2]
```diff
@@ -43,48 +45,54 @@
...
 
-@pytest.mark.parametrize("attr", scalars_list)
-def test_hdf_scalars(simulation_verysimple, attr, snapshot_np):
+@pytest.mark.parametrize("attr", SCALARS_LIST)
+def test_hdf_scalars(simulation_verysimple, attr, regression_data):
     actual = getattr(simulation_verysimple.plasma, attr)
     if hasattr(actual, "cgs"):
-        actual = actual.cgs.value
-    assert snapshot_np == actual
+        actual = np.array(actual.cgs.value)
```
Comment (Knights-Templars) [inline tardis/plasma/tests/test_hdf_plasma.py:72]: array not needed

Comment (wkerzendorf) [reply tardis/plasma/tests/test_hdf_plasma.py:72]: done

**Comment by andrewfullard** [comment]
TODO: clean up environment after this is merged

**Review by andrewfullard — APPROVED** [review]

**Review by jvshields — APPROVED** [review]

**Pull request merged**